holidays/countries/grenada.py

```
@@ -0,0 +1,208 @@
+            (OCT, 15, tr("Aunty Tek Spice Word Festival")),
+            (OCT, 16, tr("Aunty Tek Spice Word Festival")),
+            (OCT, 17, tr("Aunty Tek Spice Word Festival")),
+            (OCT, 18, tr("Aunty Tek Spice Word Festival")),
+            (OCT, 19, tr("Aunty Tek Spice Word Festival")),
+            # Camerhogne Folk Festival.
+            (DEC, 4, tr("Camerhogne Folk Festival")),
+            (DEC, 5, tr("Camerhogne Folk Festival")),
+            (DEC, 6, tr("Camerhogne Folk Festival")),
+        ),
+        2011: (DEC, 27, bank_holiday),
+        2013: (FEB, 22, bank_holiday),
```

Ruff (0.8.2)

Dictionary key literal `2013` repeated

(F601)